content/themis/languages/python/installation.md

```diff
@@ -143,3 +141,8 @@ Consider building a `.whl` and installing it into your venv instead.
     ```bash
     make prepare_tests_all test_python
     ```
+
+To remove PyThemis after using `pythemis_old_install`, run
+```bash
+sudo pip uninstall --break-system-packages pythemis
```

Comment:
🤔

Reply:
That's what we gotta do on Debian Bookworm (Python 3.11.2, pip 23.0.1) if first installed PyThemis system-wide "the old way" and then trying to remove it. Otherwise, without the suspicious option, enjoy the error:
```
$ sudo pip uninstall pythemis
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.

    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.

    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.

    See /usr/share/doc/python3.11/README.venv for more information.
```
Same error for non-`sudo` command, it just wants to be used in virtual environment. Or use the OS package (that's not done yet)